Header header
uint32 gpsDate  # GPS Date  in the format: yyyymmdd (GMT+8)
uint32 gpsTime  # GPS Time in the format: hhmmss (GMT+8)



